metaflow_extensions/utils.py

```diff
@@ -36,7 +39,7 @@ def walk(root: os.PathLike, addl_suffixes: List[str]) -> Iterator[str]:
 
 def is_path_hidden(path: os.PathLike) -> bool:
     """True if `path` contains a hidden component."""
-    return "/." in path or Path(path).name.startswith(".")
+    return "/." in str(path) or Path(path).name.startswith(".")
```

Comment:
I think more precisely:

```suggestion
    return any(map(lambda part: part.startswith('.'), Path(path).parts))
```